tests/test_upload.py

```diff
@@ -57,6 +59,104 @@ def test_successful_upload(make_settings, capsys):
     assert captured.out.count(NEW_RELEASE_URL) == 1
 
 
+def test_successful_upload_add_gpg_signature(make_settings, capsys, monkeypatch):
+    """Test uploading package when gps signature are added to it"""
```

Comment:
`... when *gpg* signatures are added to it.`

Reply:
Fixed in latest commit